deploy.sh

```
@@ -0,0 +1,44 @@
+JAR_NAME=$(ls -tr $APP_DIR/*.jar | tail -n 1)
+
+if [ -z "$JAR_NAME" ]; then
+  echo "> 오류: 배포할 JAR 파일을 찾을 수 없습니다."
+  exit 1
+fi
+
+echo "> 배포할 JAR: $JAR_NAME"
+
+if [ -f "$APP_DIR/.env" ]; then
+  echo "> .env 파일을 로드합니다."
+  export $(cat $APP_DIR/.env | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)